src/media/js/elements/nav.js

```diff
@@ -220,9 +246,8 @@ define('elements/nav',
     z.page.on('navigate', function() {
         // Close all menus. The try/catch block is necessary because navigate
         // gets triggered before the custom element is registered with the DOM.
-        var navs = document.querySelectorAll('mkt-nav');
-        for (i = 0; navs && (i < navs.length); i++) {
-            var nav = navs[i];
+        var nav = document.querySelector('mkt-nav');
+        if (nav) {
             try {
```

Comment:
Not related to this but it doesn't look like the `try`/`catch` is going to catch an error in the `setTimeout`. It shouldn't make a difference but it will get logged.